practice-one/src/components/CustomerItem/CustomerItem.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import "./CustomerItem.css";
+import { menuDot } from "@assets/images";
+import { CustomerInfo, Gender, ContextMenu } from "@components";
+
+const CustomerItem = ({ avatar, name, email, phoneNumber, gender }) => {
+  const [isShowSubMenu, setIsShowSubMenu] = useState(false);
```

Comment:
Do you understand sub menu is mean?

Reply:
Sorry, I don't  understand that, I just named it with my poor knowledge.
Can I name it is `isShowContextMenu`?